tests/fixtures/addon/with-app-styles/package.json

```diff
@@ -4,10 +4,7 @@
   "dependencies": {
     "something-else": "latest"
   },
-  "ember-addon": {
-    "paths": ["./lib/ember-super-button"]
```

Comment:
The `ember-super-button` addon does not exist for this test fixture. Not needed as well.